models/sap.yml

```diff
@@ -1,11 +1,975 @@
+      - name: bupla
+        description: Business place
+      - name: pyamt
+        description: Amount in currency for automatic payment (absolute)
+      - name: pycur
+        description: Currency for automatic payment
+      - name: cession_kz
+        description: Accounts receivable pledging indicator
+      - name: xragl
+        description: Indicator (Clearing was undone)
+      - name: auggj
+        description: Fiscal year of clearing document
+      - name: buzid
+        description: Identification of posting line
+      - name: bdiff
+        description: Valuation difference
+      - name: agzei
+        description: Clearing item
+      - name: bdif2
+        description: Valuation difference for the second local currency
+      - name: bdif3
+        description: Valuation difference for the third local currency
+      - name: dabrz
+        description: Asset value date for billing
+      - name: bewar
+        description: Flow type
+      - name: fkber
+        description: Functional area
+      - name: dmbtr
+        description: Amount in local currency
+      - name: imkey
+        description: Internal key for Real Estate object
+      - name: fkber_long
+        description: Functional area (long)
+      - name: kunnr
+        description: Customer number 1
+      - name: kstar
+        description: Cost element
+      - name: meins
+        description: Base unit of measure
+      - name: lifnr
+        description: Account number of vendor
+      - name: pargb
+        description: Business area of the business partner
+      - name: menge
+        description: Quantity
+      - name: pprct
+        description: Partner profit center
+      - name: pfkber
+        description: Functional area of partner
+      - name: wrbtr
+        description: Amount in document currency
+      - name: saknr
+        description: G/L account number
+      - name: xlgclr
+        description: Clearing specific to ledger groups
+      - name: xopvw
+        description: Indicator (Open Item Management?)
+      - name: zzspreg
+        description: Special region
+      - name: xreorg
+        description: Doc. contains open item that Was transferred during reorg.
+      - name: zzbuspartn
+        description: Business partner
+      - name: zzproduct
+        description: Product group
+      - name: zzloca
+        description: City
+      - name: zzchan
+        description: Distribution channel
+      - name: zzlob
+        description: Business line
+      - name: zzuserfld1
+        description: Territory
+      - name: zzuserfld2
+        description: Owner/Cont.
+      - name: zzuserfld3
+        description: Vein
+      - name: zzregion
+        description: User defined regions
+      - name: zzstate 
+        description: State/province code
+
   - name: sap__0gl_account_attr
-    description: Access the general ledger account number attribute.
+    description: G/L Account Number data source
```

Comment:
Request for more details in description of the model.

Reply:
Updated